Kill THIntegerTensor, THDenseTensor, THDenseIndexTensor.

gchanan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #32657 Get rid of some unused THGenerate*Type defines.
* #32600 Kill some more unused code in function_wrapper.py
* **#32599 Kill THIntegerTensor, THDenseTensor, THDenseIndexTensor.**

these aren't used anymore.

Differential Revision: [D19565655](https://our.internmc.facebook.com/intern/diff/D19565655)

gchanan committed:
Kill THIntegerTensor, THDenseTensor, THDenseIndexTensor.

these aren't used anymore.

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 37fcfd4b:


**None of the build failures appear to be your fault**.

* **1/1** recognized as flaky :snowflake:
    * Re-run these jobs?


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=37fcfd4b96b4e399079bef5681c1879ed99152d4).

### :snowflake: 1 failure recognized as flaky
The following build failures have been detected as flaky and **may not be your fault**:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4336363) pytorch_linux_xenial_cuda10_1_cudnn7_py3_gcc7_build (1/1)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=90517254) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=90517254)) :snowflake:
<details>
<summary>
<code>Jan 24 23:25:58 E: Failed to fetch https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/Packages  Operation too slow. Less than 10 bytes/sec transferred the last 120 seconds</code>
</summary>

```
Jan 24 23:23:46   libcurl            pkgs/main/linux-64::libcurl-7.67.0-h20c2e04_0 
Jan 24 23:23:46   libssh2            pkgs/main/linux-64::libssh2-1.8.2-h1ba5d50_0 
Jan 24 23:23:46   rhash              pkgs/main/linux-64::rhash-1.3.8-h1ba5d50_0 
Jan 24 23:23:46  
Jan 24 23:23:46  
Jan 24 23:23:46 Preparing transaction: ...working... done 
Jan 24 23:23:47 Verifying transaction: ...working... done 
Jan 24 23:23:47 Executing transaction: ...working... done 
Jan 24 23:23:47 + [[ pytorch-linux-xenial-cuda10.1-cudnn7-py3-gcc7-build == *-xenial-cuda10.1-* ]] 
Jan 24 23:23:47 + sudo apt-get -qq update 
Jan 24 23:25:58 E: Failed to fetch https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/Packages  Operation too slow. Less than 10 bytes/sec transferred the last 120 seconds 
Jan 24 23:25:58 E: Some index files failed to download. They have been ignored, or old ones used instead. 
Jan 24 23:25:58 + cleanup 
Jan 24 23:25:58 + retcode=100 
Jan 24 23:25:58 + set +x 
Jan 24 23:25:58 =================== sccache compilation log =================== 
Jan 24 23:25:58 =========== If your build fails, please take a look at the log above for possible reasons =========== 
Jan 24 23:25:58 Compile requests                 0 
Jan 24 23:25:58 Compile requests executed        0 
Jan 24 23:25:58 Cache hits                       0 
Jan 24 23:25:58 Cache misses                     0 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

gchanan committed:
Update on "Kill THIntegerTensor, THDenseTensor, THDenseIndexTensor."


these aren't used anymore.

Differential Revision: [D19565655](https://our.internmc.facebook.com/intern/diff/D19565655)

[ghstack-poisoned]

bhosmer reviewed (approved):
This LGTM - just to confirm, looks like these are completely unused so the PR just erases them, right? ...and if so, I found this :D https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/preprocess_declarations.py#L114

gchanan commented:
yes, they are completely unused.  Good find.
